specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -0,0 +1,1288 @@
+                    },
+                    "description": "Whether to send security alerts notifications to subscription admins"
+                }
+            },
+            "required": [
+                "email",
+                "phone",
+                "alertNotifications",
+                "alertsToAdmins"
+            ]
+        },
+        "PricingList": {
+            "description": "List of pricing configurations",
+            "required": [
+                "value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Pricing"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "Pricing": {
+            "type": "object",
+            "description": "Pricing tier will be applied for the scope based on the resource ID",
+            "properties": {
+                "properties": {
+                 "$ref": "#/definitions/PricingProperties"   
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "PricingProperties": {
+            "type": "object",
+            "properties": {
+                "pricingTier": {
+                    "type": "string",
+                    "enum": ["Free", "Standard"],
+                    "x-ms-enum": {
+                        "name": "pricingTier",
+                        "modelAsString": false
+                    }
+                }
+            },
+            "required": [
+                "pricingTier"
+            ]
+        },
+        "WorkspaceSettingList": {
+            "description": "List of workspace settings",
+            "required": [
+                "value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/WorkspaceSetting"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "WorkspaceSetting": {
+            "type": "object",
+            "description": "Configures where to store the OMS agent data for workspaces under a scope",
+            "properties": {
+                "properties": {
+                    "$ref":"#/definitions/WorkspaceSettingProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "WorkspaceSettingProperties": {
+            "readOnly": true,

```

Comment:
This property should only be applied on a property. It does not make sense to add this on a model definition.

Reply:
Removed for all occurrences